projects/cdk/utils/dom/is-inside-iframe.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Checks if an app is running inside <iframe /> tag
+ */
+export function isInsideIframe(windowRef: Window): boolean {
+    return windowRef.parent !== windowRef;
```

Comment:
I chose this method of detection that app is running under iframe because it allow to manually configure it inside tests (`windowRef.parent` can be force overwritten).